var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -37,14 +37,16 @@ class Lammps(CMakePackage):
     url      = "https://github.com/lammps/lammps/archive/patch_1Sep2017.tar.gz"
 
     version('20170901', '767e7f07289663f033474dfe974974e7')
+    version('20170922', '4306071f919ec7e759bda195c26cfd9a')
```

Comment:
we usually put them in descending order (sometimes with exception of `develop` being at the very bottom).

Reply:
Fixed!